README.md

```diff
@@ -1,5 +1,41 @@
 # Ruby_TicTacToe
 
-# Project: TicTacToe
+# Tic Tac Toe OOP (Object Oriented Programming)
 
-Using Ruby OOP (Object Oriented Programming) Approach
+-   Original project specification at [The Odin Project](https://www.theodinproject.com/courses/ruby-programming/lessons/oop)
+-   Tic-Tac-Toe [From Wikipedia](https://en.wikipedia.org/wiki/Tic-tac-toe) for more details about Tic-Tac-Toe.
```

Comment:
it's better to add the rules of the game without using links to them since they are short and it makes the document easier to read.